applications/cambiatus.md

```diff
@@ -211,13 +215,13 @@ Registered Address: PO Box 2510, Grand Cayman KY1-1104, Cayman Islands
 
 
 ### Team Experience
-Our team has been building on EOSIO for 4 years now. We've done our pilot in a community in Costa Rica. We've been part of every major announcement since before Block.one released version 1.0. Since then we've deployed and maintained our own private network, developed several features into our app, including smart contract, backend, infrastructure, wallet integrations, frontend and finally our own wallet inside our app.
+Our team has been building on EOSIO for 4 years now. We've done our pilot in a [community in Costa Rica](https://www.cambiatus.com/verdes), which is still ongoing. We've been part of every major announcement since before Block.one released version 1.0. Since then we've deployed and maintained our own private network, developed several features into our app, including [smart contract](https://github.com/cambiatus/contracts), [backend](https://github.com/cambiatus/backend), [infrastructure](https://github.com/cambiatus/backend#nginx-setup), wallet integrations, [frontend](https://github.com/cambiatus/frontend) and finally our own wallet inside our app.
 
-Although we are like projects like Hypha, Cambiatus is different because the initial focus is on social currencies and financial resiliency to foster planetary regeneration. Our partner communities are located in Latin America and Africa. We’ve had many learnings along this way, starting from simple things like choosing to develop a mobile-first webApp because many of our members didn’t have enough memory space in their smartphones to download an app, access to a PC/laptop or have a slow and unreliable internet connection. 
+Although we have similarities with projects like Hypha, Cambiatus is different because our main focus is on social currencies and financial resiliency to foster planetary regeneration. This is shown through our actions, such as choosing to develop a mobile-first webApp because many of our members don’t have enough memory space in their smartphones to download an app, others don't have access to a PC/laptop and others yet have a slow and unreliable internet connection. These actions are reflections of our goals and learnings with our partner communities which are located in Latin America and Africa.
```

Comment:
```suggestion
Although we admire and like projects like Hypha, Cambiatus is different because our main focus is on social currencies and financial resiliency to foster planetary regeneration. This is shown through our actions, such as choosing to develop a mobile-first webApp because many of our members don’t have enough memory space in their smartphones to download an app, others don't have access to a PC/laptop and others yet have a slow and unreliable internet connection. These actions are reflections of our goals and learnings with our partner communities which are located in Latin America and Africa.
```

Reply:
a ideia era transmitir que gostamos e apreciamos o Hypha

Reply:
Show então!